AUTHORS.md

```diff
@@ -1,4 +1,6 @@
-All of the people who have made at least one contribution to conda-package-handling.
+All of the people who have made at least one contribution to conda-package-handling
+
+.
```

Comment:
Does rever think the project name has newlines at the end? Do not want to edit this manually.